distribution/docs/src/main/resources/content/_coreConcepts/data-standards.adoc

```diff
@@ -0,0 +1,77 @@
+:type: subCoreConcept
+:status: published
+:title: Data Formats
+:parent: Standards Supported by ${branding}
+:order: 02
+
+== {title}
+
+${branding} has extended capabilities to extract rich metadata from many common data formats, if those attributes are populated in the source document.
+See <<_all_formats_supported,appendix>> for a complete list of file formats that can be ingested, with limited metadata coverage.
+
+.Data Formats Included in ${ddf-branding}
+[cols="1,1,2" options="header"]
+|===
+
+|Format
+|Versions
+|Additional Metadata Attributes Available (if populated)
+
+|Word Document
+|doc, docx, dotx, docm
+|<<_file_formats_supported,Standard attributes>>
+
+|PowerPoint
+|ppt, pptx
+|<<_file_formats_supported,Standard attributes>>
+
+|Excel
+|xls, xlsx
+|<<_file_formats_supported,Standard attributes>>
+
+|PDF
+|any
+|<<_file_formats_supported,Standard attributes>>
+
+|GeoPDF
+|any
+|<<_file_formats_supported,Standard attributes>>
+
+|html
+|5
+|<<_file_formats_supported,Standard attributes>>
+
+|xml
+|any
+|<<_file_formats_supported,Standard attributes>>
+
+|JSON
+|
+|<<_file_formats_supported,Standard attributes>>
+
+|jpeg
+|jpeg, jpeg2000
+|<<_file_formats_supported,Standard attributes>> and additional Media attributes
+
+|mp2
+|h.264 MPEG2
+|<<_file_formats_supported,Standard attributes>> and additional Media attributes
+
+|mp4
+|
+|<<_file_formats_supported,Standard attributes>>, additional Media attributes, and <<_mp4_additional_attribute,mp4 additional attribute>>
+
+|Atlassian Confluence(R)
```

Comment:
❓  A little confused by what we mean by this one

Reply:
It is an odd fit in either list